packages/main/src/requests/request.ts

```diff
@@ -107,6 +107,9 @@ export async function makeRequest(
  */
 function buildFetchOptions(requestOptions?: RequestOptions): RequestInit {
   const fetchOptions = {} as RequestInit;
+  if (requestOptions?.signal) {
+    fetchOptions.signal = requestOptions?.signal;
```

Comment:
the optional chaining can be omitted

Reply:
Ok, https://github.com/google/generative-ai-js/pull/50/commits/904795ee86436348cccf34d9ca8c761500135771